src/index.html

```diff
@@ -3,11 +3,287 @@
+          </p>
+          <p class="section__price">
+            $ 149.00
+          </p>
+        </article>
+        <article class="section__article section__article--main">
+          <img
+            class="section__picture"
+            src="images/section3.jpg"
+            alt="phone2">
+          <h3 class="section__title">
+            Ear (stick)
+          </h3>
+          <p class="section__paragraph section__paragraph--main">
+            Tech you can &#x2019; feel. Leaving just space for sound to come to life. Powered by custom sound technology.
+          </p>
+          <p class="section__price">
+            $ 119.00
+          </p>
+        </article>
+      </section>
+
+      <h2 class="title title__secondiary title__secondiary--browse main__title" id="store">
+        Browse Nothing products by category
+      </h2>
+
+      <section class="section main__section">
+        <article class="section__article">
+
+          <div class="pictures">
+            <img
+              class="pictures__item pictures__item--big"
+              src="images/all-products1.jpg"
+              alt="birds-on-phone"
+            >
+            <img
+              class="pictures__item pictures__item--small"
+              src="images/all-products2.jpg"
+              alt="headphones-box-1"
+            >
+            <h3 class="section__title section__title--second">
+              All products
+            </h3>
+          </div>
+
+
+        </article>
+
+        <article class="section__article">
+
+          <div class="pictures">
+            <img
+              class="pictures__item pictures__item--small"
+              src="images/audio1.jpg"
+              alt="headphones"
+            >
+            <img
+              class="pictures__item pictures__item--big"
+              src="images/audio2.jpg"
+              alt="headphones-box-1"
+            >
+
+            <h3 class="section__title section__title--second">
+              Audio
+            </h3>
+          </div>
+
+
+        </article>
+
+        <article class="section__article">
+
+          <div class="pictures">
+            <img
+              class="pictures__item pictures__item--big"
+              src="images/accessories1.jpg"
+              alt="mobilephone"
+            >
+            <img
+              class="pictures__item pictures__item--small"
+              src="images/accessories2.jpg"
+              alt="phone-charger"
+            >
+
+            <h3 class="section__title section__title--second section__title--button">
+              Accessories
+            </h3>
+          </div>
+
+          <a href="#" class="button">View all</a>
+        </article>
+      </section>
+    </main>
+
+    <div class="separator"></div>
+
+    <footer class="footer">
+      <div class="about">
+        <section class="section">
+          <h2 class="title title__secondiary title__about">
```

Comment:
'title__secondiary' used as a class name seems to be misspelled. Correct spelling should be 'title__secondary'.